client/components/NewMessage.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+
+import people from '../people.js'
+
+const styles = {
+  borderRadius: '150px',
+  background: 'white',
+  width: '300px',
+  height: '300px'
+}
+
+class NewMessage extends React.Component {
+  constructor () {
+    super()
+    this.state = {
+      sender: null,
+      recipient: null,
+      senderPhoto: null,
+      recipientPhoto: null
+    }
+    this.handleChange = this.handleChange.bind(this)
+    this.handleGetPhoto = this.handleGetPhoto.bind(this)
+  }
+  handleGetPhoto (action, name) {
+    if (action === 'sender') {
+      people.map((person) => {
+        if (person.name === name) {
+          this.setState({senderPhoto: person.photo})
+        }
+      })
+    } else if (action === 'recipient') {
+      people.map((person) => {
+        if (person.name === name) {
+          this.setState({recipientPhoto: person.photo})
+        }
+      })
+    }
+  }
+
+  handleChange (e) {
+    const target = e.target
+    const name = target.value
+    const action = target.name
+
+    this.setState({
+      [action]: name
+    })
+    this.handleGetPhoto(action, name)
+  }
+  render () {
+    return (
+      <div>
+        <form>
+              <div>
+                <select name="sender" onChange={this.handleChange}>
+                <option value="sender">Sender</option>
+                  {people.map((person) => {
+                    return (
+                      <option key={person.id} value={person.name}>{person.name}</option>
```

Comment:
I think using the `person.id` would be better for the `value` attribute. When we get to subitting this to the API, we'll be sending the persons `id` rather than their `name`.